tests/helpers/mender-configure-server.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import http.server as server
+import os
+
+class MenderConfigureHandler(server.BaseHTTPRequestHandler):
+    def do_PUT(self):
+        auth = self.headers.get("Authorization", "")
+
+        if self.path != "/configuration":
+            response = 404
+        elif auth != "Bearer ThisIsAnAuthToken":
+            response = 401
+        else:
+            response = 204
+        self.send_response(response)
+
+        with open(os.environ["TEST_HTTP_LOG"], "a") as log:
```

Comment:
What if this env variable is not set? I can imagine running this server manually while debugging something.

Reply:
Alright, I made a default.